### DISCUSSION TITLE
Can I generate just types without a client?

### DISCUSSION THREAD

**Discussion by leo-petrucci** [Azure/autorest#4542] [Q&A]
I'm planning to use `autorest.typescript` to automate type generation from my .NET Rest API, so far it looks fantastic and works like a charm but I'm seeing a lot of code generated that I don't really need, like the API client for example.

Is there any way to just generate interfaces and types?

**Comment by raky291** [comment]
Same question here, is it possible?

@leo-petrucci did you find something?
